domino/routes.py

```diff
@@ -150,6 +150,12 @@ def app_get(self, app_id):
         return f'{self.host}/v4/modelProducts/{app_id}'
 
     # Hardware Tier URLs
+    def hardware_tier_create(self, hardware_tier_id, hardware_tier_name, node_pool):
+        return self.host + f'/admin/hwtiers/insert'
```

Comment:
Can admin endpoints only be called by admin users?

Reply:
yes that should be the case

if not it is a bug